docs/audit-log-filter/install-audit-log-filter.md

```diff
@@ -0,0 +1,30 @@
+# Install the Audit Log Filter
+
+The `plugin_dir` system variable defines the plugin library location. If needed, at server startup, set the `plugin_dir` variable.
+
+When upgrading a MySQL installation, plugins are not automatically upgraded. You may need to manually load certain functions or reinstall the plugin after the MySQL upgrade.
+
+In the `share` directory, locate the `audit_log_filter_linux_install.sql `script.
+
+```{.bash data-prompt="$"}
+$ mysql -u root -p < audit_log_filter_linux_install.sql
```

Comment:
We need to update this according to latest changes at https://dev.mysql.com/doc/refman/8.0/en/audit-log-installation.html. Please take a look at MySQL 8.0.34 related additions. We should use the following to install the plugin

$mysql -u root -D database_name -p < audit_log_filter_linux_install.sql

Reply:
added.